app/batid/services/data_gouv_publication.py

```diff
@@ -50,12 +50,44 @@ def file_path(directory_name, code_area):
     return directory_name + "/RNB_" + str(code_area)
 
 
+def sql_query(code_area):
+    if code_area == "nat":
+        filter_condition = ""
+    else:
+        filter_condition = f"WHERE code_dept = '{code_area}'"
+
+    sql = f"""
```

Comment:
J'ajoute une idée d'optimisation de perf si un jour on veut accélérer l'export.

```suggestion
    # If one day, we want to optimize performance:
    # we could do only one query to get all the national data and then split this data into departments files using the `code_dept` instead of doing 98 or more queries
    sql = f"""
```